<launch>
    <arg name="visualization" default="false"/>
    <!-- <arg name="location" default="gotthard"/> -->
    <!-- <arg name="location" default="dischma_valley"/> -->
    <arg name="location" default="sargans"/>

    <node pkg="tf" type="static_transform_publisher" name="world_map" args="0 0 0 0 0 0 world map 10"/>

    <node pkg="terrain_planner_benchmark" type="test_ics" name="terrain_planner" output="screen" required="true">
        <param name="visualize" value="$(arg visualization)"/>
        <param name="location" value="$(arg location)"/>
        <param name="map_path" value="$(find terrain_models)/models/$(arg location).tif"/>
        <param name="color_file_path" value="$(find terrain_models)/models/$(arg location)_color.tif"/>
        <param name="output_file_dir" value="$(find terrain_planner)/../output"/>
    </node>
    <group if="$(arg visualization)">
        <node type="rviz" name="rviz" pkg="rviz" args="-d $(find terrain_planner)/launch/config_ics.rviz" />
    </group>
</launch>
